recipes/recipes_emscripten/xeus_shared/patches/shared.patch

```diff
@@ -0,0 +1,27 @@
+diff --git a/CMakeLists.txt b/CMakeLists.txt
+index a8147d3..f3a99c6 100644
+--- a/CMakeLists.txt
++++ b/CMakeLists.txt
+@@ -59,8 +59,12 @@ option(XEUS_BUILD_TESTS "xeus test suite" OFF)
+ # ===================================
+ 
+ if(EMSCRIPTEN)
+-    SET(XEUS_BUILD_SHARED_LIBS OFF)
+-    SET(XEUS_BUILD_STATIC_LIBS ON)
++    SET(XEUS_BUILD_SHARED_LIBS ON)
++    SET(XEUS_BUILD_STATIC_LIBS OFF)
```

Comment:
I think we should rather completely remove the `SET(XEUS_BUILD_SHARED_LIBS` and `SET(XEUS_BUILD_STATIC_LIBS` instructions, and pass the options command line. After all, these are command line options, and this would make the emscripten case less special.